### ISSUE TITLE
stylegan3 encoder

### ISSUE THREAD

**Issue by soushirou** [eladrich/pixel2style2pixel#235]
Thank you for your great work, 
Using your psp architecture, I can train stylegan3(Alias Free GAN)'s encoder and get reasonable results.

Loss, hyperparameters, and optimizers are same as your configuration, 
some are changed, 
batch size(from 4 to 32) and using DDP to train
model architecture, from 18 GradualStyleBlock to 16 

here is my implementation link
https://github.com/soushirou/stylegan3-encoder

**Comment by yuval-alaluf** [comment]
Hi @soushirou 
Looks really cool! I'll be sure to keep an eye out for updates. Your TODO looks exciting! 
Could we add a link to your repo in our README?

**Comment by soushirou** [comment]
I appreciate it if you add my implementation to your repo:)
I'll do experiments related over parametrization and e4e loss this week.
Thank you.
